README.md

```diff
@@ -163,6 +163,7 @@ The `{% main_menu %}` tag allows you to display the `MainMenu` defined for the c
 - **`apply_active_classes`** (default: `True`): The tag will attempt to add 'active' and 'ancestor' CSS classes to the menu items (where applicable) to indicate the active page and ancestors of that page. To disable this behaviour, add `apply_active_classes=False` to the tag in your template. You can change the CSS classes used by adding `WAGTAILMENUS_ACTIVE_CLASS` and `WAGTAILMENUS_ACTIVE_ANCESTOR_CLASS` settings to your project's settings module.
 - **`template`** (default: `''`): Lets you render the menu to a template of your choosing. If not provided, wagtailmenus will look in several locations for an appropriate template (see below for details).
 - **`sub_menu_template`** (default: `''`): Lets you specify a template to be used for rendering sub menus. All subsequent calls to `{% sub_menu %}` within the context of the section menu will use this template unless overridden by providing a `template` value to `{% sub_menu %}` in a menu template. If not provided, wagtailmenus will look in several locations for an appropriate template (see below for details).
+- **`use_absolute_urls`** (default: `False`): Will use absolute URLs when creating the links for pages.
```

Comment:
Sorry, I don't think I was clear enough in my explaination there. it looks as though you've left the reference to `{% main_menu %}` the same for each tag. That needs to change for each tag the instructions are for.